CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -304,8 +314,7 @@ function Wait-RemoteInterfaceIP {
         Invoke-Command -Session $Session {
             Get-NetAdapter -Name $Using:AdapterName `
                 | Get-NetIPAddress -ErrorAction SilentlyContinue `
-                | Where-Object AddressFamily -eq IPv4 `
-                | Where-Object { ($_.SuffixOrigin -eq "Dhcp") -or ($_.SuffixOrigin -eq "Manual") }
+                | Select-ValidNetIPInterface
```

Comment:
You can try the following approach:
```powershell
$Body = (Get-Item function:Select-ValidNetIPInterface).Definition
Invoke-Command -Session $Session {
    function Select-ValidNetIPInterface {
        & [Sriptblock]::Create($Using:Body)
    }
    ...
}
```

But perhaps that's an overkill and another place for errors to occur.

Perhaps we could make a wrapper:
```powershell
Invoke-CommandWithFunctions -Session $Session `
    -Functions "Select-ValidNetIPInterface" { ... }
```
that would handle copying functions to remote session for us and be properly tested. But perhaps that's even bigger overkill.